tests/aws/services/ec2/test_ec2.py

```diff
@@ -719,10 +722,21 @@ def test_get_security_groups_for_vpc(
         snapshot.add_transformer(snapshot.transform.key_value("VpcId"))
         snapshot.add_transformer(snapshot.transform.key_value("AssociationId"))
         snapshot.add_transformer(snapshot.transform.key_value("DhcpOptionsId"))
+        snapshot.add_transformer(snapshot.transform.key_value("Key"))
+        snapshot.add_transformer(snapshot.transform.key_value("Value"))
 
         # Create VPC for testing
         vpc: dict = create_vpc(
             cidr_block="10.0.0.0/16",
+            tag_specifications=[
+                {
+                    "ResourceType": "vpc",
+                    "Tags": [
+                        {"Key": "Name", "Value": vpc_name},
```

Comment:
Try to avoid the value `Name` in `Key` attribute because it seems to mess up the snapshots.

```suggestion
                        {"Key": "SomethingElse", "Value": vpc_name},
```

Reply:
I've opted to just remove this tag since another test tag is already being created so it's not actually necessary